From adcfeba71f5e5ad899102ff5675c281b2366edb9 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@nxp.com>
Date: Thu, 26 May 2016 16:56:01 +0800
Subject: [PATCH 1075/5242] MLK-12860-1 ARM: dts: imx7d: add vcc-supply at gpc
 node

commit  e5c046378213e02e41475763765167ea78c474f1 from
https://source.codeaurora.org/external/imx/linux-imx.git

It is missing at imx7d.dtsi, but used at source code.

Signed-off-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 1e34ffc..076dc00 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -189,6 +189,7 @@
 		fsl,mf-mix-wakeup-irq = <0x54010000 0xc00 0x0 0x1040640>;
 		mipi-phy-supply = <&reg_1p0d>;
 		pcie-phy-supply = <&reg_1p0d>;
+		vcc-supply = <&reg_1p2>;
 	};
 
 	iomuxc_lpsr_gpr: lpsr-gpr@30270000 {
-- 
1.7.9.5

